src/aro/Type.zig

```diff
@@ -1899,6 +1899,7 @@ pub const Builder = struct {
 
     /// Try to combine type from typedef, returns true if successful.
     pub fn combineTypedef(b: *Builder, p: *Parser, typedef_ty: Type, name_tok: TokenIndex) bool {
+        if (typedef_ty.is(.invalid)) return false;
```

Comment:
I'm guessing this returns false so that the typedef name token is used as the declaration name instead of another type specifier? Merge to confirm.